insecure/corruptible/factory.go

```diff
@@ -297,11 +328,21 @@ func (c *ConduitFactory) sendOnNetwork(event interface{},
 		return c.adapter.UnicastOnChannel(channel, event, targetIds[0])
 
 	case insecure.Protocol_PUBLISH:
-		return c.adapter.PublishOnChannel(channel, event)
+		return c.adapter.PublishOnChannel(channel, event, targetIds...)
 
 	case insecure.Protocol_MULTICAST:
 		return c.adapter.MulticastOnChannel(channel, event, num, targetIds...)
 	default:
 		return fmt.Errorf("unknown protocol for sending on network: %d", protocol)
 	}
 }
+
+func (c *ConduitFactory) generateExecutionReceipt(result *flow.ExecutionResult) (*flow.ExecutionReceipt, error) {
+	// TODO: fill spock secret with dictated spock data from attacker.
```

Comment:
Note that current state of Flow Protocol does not check SPOCK proofs generated by ENs and VNs, hence, we leave the SPOCK secret empty for now, as enabling CCF to generate SPOCK proof needs extra efforts which is out of the scope of this PR. The wintermute attack should work with empty SPOCK proof.